<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="login_frame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>120</y>
      <width>500</width>
      <height>205</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(255, 255, 255);
	border-top-left-radius: 20px;
	border-top-right-radius: 20px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="account_edit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-top: none;
	border-bottom: 1px solid rgb(187, 187, 187);
	padding-left: 30px;
}

QLineEdit:hover {
	border-bottom: 1px solid rgb(220, 195, 170);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>账号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_btn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>135</y>
       <width>300</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: none;
	color: rgb(53, 53, 53);
	border-radius: 15px;
	border: 1px solid rgb(220, 195, 170);
}

QPushButton:hover {
	background-color: rgb(220, 195, 170);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	padding-left: 1px;
	padding-top: 1px;
}</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_edit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-top: none;
	border-bottom: 1px solid rgb(187, 187, 187);
	padding-left: 30px;
}

QLineEdit:hover {
	border-bottom: 1px solid rgb(220, 195, 170);
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../PycharmProjects/Status/resources/icon/login/account.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../PycharmProjects/Status/resources/icon/login/password.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="student_checkBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>107</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	color: rgb(166, 166, 166);
}</string>
     </property>
     <property name="text">
      <string>学生</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QCheckBox" name="teacher_checkBox">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>107</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox {
	color: rgb(166, 166, 166);
}</string>
     </property>
     <property name="text">
      <string>教师</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="warn_text">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>175</y>
       <width>300</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>500</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(37, 40, 43);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>60</width>
       <height>45</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>关闭</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: none;
}

QPushButton:hover {
	background-color: rgb(244, 84, 84);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../PycharmProjects/Status/resources/icon/main/close.png</normaloff>../../../../../PycharmProjects/Status/resources/icon/main/close.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="minus_btn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>60</width>
       <height>45</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>最小化</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: none;
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../PycharmProjects/Status/resources/icon/main/minus.png</normaloff>../../../../../PycharmProjects/Status/resources/icon/main/minus.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="logo_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>8</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../PycharmProjects/Status/resources/pic/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="loading_label_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>115</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>52</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background: none;</string>
     </property>
     <property name="text">
      <string>·</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="loading_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>115</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>52</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background: none;</string>
     </property>
     <property name="text">
      <string>·</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="loading_label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>115</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>52</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background: none;</string>
     </property>
     <property name="text">
      <string>·</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="shadow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>510</width>
      <height>330</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../PycharmProjects/Status/resources/pic/login_win_shadow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>shadow</zorder>
   <zorder>frame_2</zorder>
   <zorder>login_frame</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>account_edit</tabstop>
  <tabstop>password_edit</tabstop>
  <tabstop>student_checkBox</tabstop>
  <tabstop>teacher_checkBox</tabstop>
  <tabstop>confirm_btn</tabstop>
  <tabstop>minus_btn</tabstop>
  <tabstop>close_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minus_btn</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
